frontend/config-assistant/src/components/gui-editor/PropertiesPanel.vue

```diff
@@ -98,6 +99,12 @@ function updateData(subPath: Path, newValue: any) {
   const completePath = props.currentPath.concat(subPath);
   emit('update_data', completePath, newValue);
 }
+function clickedPropertyData(nodeData: ConfigTreeNodeData) {
+  const path = nodeData.absolutePath;
+  if (useSessionStore().dataAtPath(path) != undefined) {
```

Comment:
you use the session store to access the data at the path, but then don't use the session store to update the selection. What is the reason behind this ? seems inconsistent